flytekit/bin/entrypoint.py

```diff
@@ -110,5 +114,39 @@ def execute_task_cmd(task_module, task_name, inputs, output_prefix, raw_output_d
     _execute_task(task_module, task_name, inputs, output_prefix, raw_output_data_prefix, test)
 
 
+@_pass_through.command("pyflyte-fast-execute")
+@_click.option("--task-module", required=True)
+@_click.option("--task-name", required=True)
+@_click.option("--inputs", required=True)
+@_click.option("--output-prefix", required=True)
+@_click.option("--raw-output-data-prefix", required=False)
+@_click.option("--additional-distribution", required=False)
+@_click.option("--test", is_flag=True)
+def fast_execute_task_cmd(
+    task_module, task_name, inputs, output_prefix, raw_output_data_prefix, additional_distribution, test
+):
+    if additional_distribution is not None:
+        _download_distribution(additional_distribution, _pathlib.Path(_os.getcwd()))
+
+    # Use the commandline to run the task execute command rather than calling it directly in python code
+    # since the current runtime bytecode references the older user code, rather than the downloaded distribution.
+    cmd = _sdk_config.SDK_PYTHON_VENV.get() + [
```

Comment:
I think this is the problem, maybe we should have this as an argument to the fast execute command? This is because we may change this right?

Reply:
updated